packages/marble-api/openapis/marblecore-api.yaml

```
@@ -4274,6 +4275,40 @@ components:
             user_id:
               type: string
               format: uuid
+    DecisionReviewedEventDto:
+      allOf:
+        - $ref: '#/components/schemas/CaseEventDto'
+        - type: object
+          required:
+            - additional_note
+            - new_value
+            - previous_value
+            - resource_id
+            - resource_type
+            - user_id
+          properties:
+            additional_note:
+              type: string
+              description: The note added by the user to justify the review
+            new_value:
+              type: string
+              format: uuid
+              description: The new review_status of the decision
+              enum: ['approve', 'decline']
+            previous_value:
+              type: string
+              format: uuid
+              description: The previous review_status of the decision
+            resource_id:
+              type: string
+              format: uuid
+              description: The id of the decision being reviewed
+            resource_type:
+              type: string
+              description: Must take the value 'decision'
```

If it will always be `'decision'` in that case, you can improve this by doing 
```suggestion
            resource_type:
              type: string
              enum: ['decision']
```

---

Right 👍 
It's a bit awkward to do an enum with just one option, but I guess that's a side effect of how we split the event into different possible options in the openapi, and it serves to document the values just as well !